hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/acl/OMBucketAclRequest.java

```diff
@@ -136,7 +136,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
     } catch (IOException ex) {
       exception = ex;
-      omClientResponse = onFailure(omResponse, ex);
+      omClientResponse = onFailure(omResponse, exception);
```

Comment:
Can we revert this change? Also many of below request java files, which change the "ex" to "exception".